DTR.md

```diff
@@ -0,0 +1,39 @@
+Project: BYOB
+
+Group Member Names: Derek Hanciles, mike duke
+
+Goals and Expectations for the Project (What does each group member hope to get out of this project? What do we want to achieve as a team? How will we know that we're successful?): 
+  - Mike's Goals: 
+    - deepen knowledge of server-side code by adding testing and getting more practice with express and knex. 
+    - get better with creating documentation for projects through README's
+
+Team strengths & collaboration styles (consider discussing your Pairin qualities here):
+   - Mike:
+     - I'm feeling pretty good about the backend code that we've learned thus far. I want to remain in contact and work collaboratively to create the best project that we can in the time we have
+
+How we can use our strengths to overcome obstacles:
+   - Mike: 
+    - i belive that Derek and i work very well together and have grown to trust each other through working on other projects together. We know where our strengths are and have no problems asking each other for help or guidance on areas of difficulty
+
+Schedule Expectations (When are we available to work together and individually? What constraints do we have?):
+  - Mike: 
+    - i know we were both planning on attending a job meetup on Wednesday night, and i assume we will work sparingly and remotely on Sunday. Any other days are open for me. 
+
+Communication Expectations (How and often will we communicate? How do we keep lines of communication open? How will we make decisions as a team?):
+  - We stay in contact over Slack very well and we have several in-class days to work on this. 
+
+Abilities & Growth Expectations (Technical strengths and areas for desired improvement):
+  - Mike: 
+    - I hope to get a solid understanding of what server-side testing looks like as well as more professional documentation. I am also looking forward to getting a first glace at how Travis CI works
+
+Workload Expectations (What features do we each want to work on?):
+  - As of Tuesday, we have divided the server routes evenly between us and are working separately to write the routes and the tests. Tomorrow we will setup and configure our databases together. I plan to take a big chunk of the README documentation. We can divvy up from there depending on time and other commitments. 
+
+Workflow Expectations (Git workflow/Tools/Code Review/Reviewing Pull Requests/Debugging and Problem-solving Techniques):
+  - We are planning on using git rebase and integrating a PR template into our workflow. We have already defined several git issues and will use those to track our progress as we move forward.
+
+Expectations for giving and receiving feedback:
+   - We are implementing a template for pull requests and are planning on doing full code reviews on each other's pull requests.
+
+Project management tools we will use:
+  - We are using git issues. 
```

Comment:
Thanks for getting the DTR added into the repo! I'll check out to a branch and add some comments as well.